src/navigator/main-stack.tsx

```diff
@@ -76,6 +77,7 @@ export const MainStackScreen = () => {
   const initDataLoading = () => {
     dispatch(loadTezosBalanceActions.submit());
     dispatch(loadTokensActions.submit());
+    dispatch(loadWhitelistAction.submit());
```

Comment:
I think this event should be dispatched once the app is loaded, there is no need in refetch each 30 sec